src/main/resources/run-docker-container.sh

```diff
@@ -3,6 +3,7 @@
 exec java \
     -DLOG_LEVEL=${LOG_LEVEL} \
     -Ds3proxy.endpoint=http://0.0.0.0:80 \
+    -Ds3proxy.virtual-host=${S3PROXY_VIRTUALHOST} \
```

Comment:
our builds using the s3proxy dockerhub image started breaking when this change was introduced..

maybe `virtual-host` used to be null? is there any way for it to still be null if it's included in the command line like this?